ui/public/i18n/en/translation.json

```diff
@@ -44,7 +44,8 @@
     "admin_password": "Password for user 'admin'",
     "domain_tooltip": "A user domain is used to authenticate users. If none user domain is listed you probably need to install one",
```

Comment:
I do not know, what do you think @andre8244 

the user will only see the `Nextcloud internal user database`, it will make an issue that we could answer by a tooltip, install an account provider

I agree to reformulate the tooltip

Reply:
OK lets remove it